corehq/apps/domain/views/settings.py

```diff
@@ -469,6 +469,23 @@ def update_feature(self, feature, current_state, new_state):
 class CustomPasswordResetView(PasswordResetConfirmView):
     urlname = "password_reset_confirm"
 
+    def get_success_url(self):
+        if self.user:
+            # redirect mobile worker password reset to a domain-specific login with their username already set
+            couch_user = CouchUser.get_by_username(self.user.username)
+            if isinstance(couch_user, CommCareUser):
```

Comment:
Nitpick: isn't it more typical to use `is_commcare_user`?

Reply:
wasn't aware of that, will change in a follow up

Reply:
I thought i'd try to figure out when this got added since I couldn't remember ever using it and figured it must have been semi-recent.

of course it was me 5 years ago https://github.com/dimagi/commcare-hq/commit/e6403c3a14a823ebf3761dbe0c6b222f6a5639d6